skbase/tests/test_base.py

```diff
@@ -1,5 +1,8 @@
 # -*- coding: utf-8 -*-
 # copyright: skbase developers, BSD-3-Clause License (see LICENSE file)
+# Elements of these tests re-use code developed in scikit-learn. These elements
```

Comment:
should this also be added in sktime?

Reply:
> should this also be added in sktime?

Yes, I think having that probably makes sense as some of the sktime tests are also sklearn inspired.

Reply:
opened an issue here in `sktime`: https://github.com/sktime/sktime/issues/3853